src/mixins/filing-mixin.ts

```diff
@@ -160,8 +165,9 @@ export default class FilingMixin extends DateMixin {
    * Builds a Restoration filing body.
    * @returns the filing body
    */
-  buildRestorationFiling (): RestorationFilingIF {
-    const restorationFiling: RestorationFilingIF = {
+  buildRestorationFiling (restorationType: RestorationTypes = RestorationTypes.FULL): RestorationFilingIF {
```

Comment:
Some linters (or maybe our linter with specific options) will complain about "redundant typing" ... it can infer that `restorationType` is of type RestorationTypes by the default value you're assigning.

Similarly, this can cause warnings:
`const myBool: boolean = true`

So, as a general rule, I don't specific the type when it can be inferred.

(Background: a while back I experimented with bringing this code base into Vue3, which brought along extra linting features. Fixing such things now should make less work in the future.)

Reply:
Ah, good point.  I'll make sure I watch for that moving forward.